### DISCUSSION TITLE
Inactive cluster

### DISCUSSION THREAD

**Discussion by urbaman** [kubescape/kubescape#1408] [Q&A] [answered]
Hi,

We had some network/stability problems qith our cluster, and now it is "inactive" on the ARMO platform. Is there a way to re-activate it?

Thank you.

**Comment by matthyx** [comment, marked as answer]
Hi @urbaman make sure all components are running, especially `gateway`, `kollector` and `operator`.

**Comment by urbaman** [comment]
Hi,

Thay were all running, but probably a network issue remained after the instability moment.
Restarting them solved, now it shows as connected, thank you very much!
